### PR TITLE
add compat data for Client#frameType and Client#postMessage

### PR DESCRIPTION
**Pull request by gsouquet**
That should complete #980

### CODE REVIEW

File: api/Client.json [hunk 1]
```diff
@@ -48,6 +48,52 @@
...
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Client.json]: Both versions of Opera should be 30.

File: api/Client.json [hunk 2]
```diff
@@ -48,6 +48,52 @@
           "deprecated": false
         }
       },
+      "frameType": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Client/frameType",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/Client.json]: This should be 43.

Comment (gsouquet) [reply api/Client.json]: The Android section on the frameType page specifies not supported. However Chrome for android says version 43.
Do you still want me to go ahead and make the change? 
I believe that Chrome for Android has its own entry in the __compat object

File: api/Client.json (hunk 1 above)
Comment (gsouquet) [reply api/Client.json]: If I look up the Client#frameType page on MDN Opera states support has unknown? Not too sure what data source to trust?

Happy to make changes if needed. Just wanna double check

Comment (teoli2003) [reply api/Client.json]: @gsouquet: Yes, @jpmedley is our human reference for Chromia like Opera :-)

File: api/Client.json (hunk 2 above)
Comment (teoli2003) [reply api/Client.json]: @gsouquet: @jpmedley is the ultimate reference for Chrome/Webview Android/Chrome for Android.

**Comment by gsouquet** [comment]
All comments fixed! Thanks @jpmedley for the feedback

**Comment by gsouquet** [comment]
@Elchi3 done, I believe I might have run `npm install` with a different `node` version which resulted in a different output for the `package-lock.json` file.
To prevent this, should we have an `engine` entry in the `package.json` https://docs.npmjs.com/files/package.json#engines or maybe add a `.nvmrc` file?

**Comment by Elchi3** [comment]
@gsouquet I'm not sure. Maybe it's me who needs to update to a newer node or npm version? I will double check. In any case, should be discussed in a new issue if this is a problem.

**Pull request merged**